kvstore/src/main/scala/zio/kvstore/App.scala

```diff
@@ -31,31 +32,30 @@ object KVCommand:
     .typecase("S", setCodec)
     .typecase("G", getCodec)
 
-class KVStateMachine(map: Map[String, String]) extends StateMachine[KVCommand]:
+class KVStateMachine(map: Map[String, String]) extends StateMachine[Map[String, String], Nothing, KVCommand]:
```

Comment:
```suggestion
class KVStateMachine() extends StateMachine[Map[String, String], Nothing, KVCommand]:
```

Reply:
why remove it? this is the initial state of the test state machine no?